src/v/cluster/feature_manager.cc

```diff
@@ -297,16 +303,14 @@ feature_manager::auto_activate_features(
          *     is satisfied by cluster's original version.
          */
         for (const auto& fs : _feature_table.local().get_feature_state()) {
-            if (
-              (fs.get_state() == features::feature_state::state::unavailable
-               || fs.get_state() == features::feature_state::state::available)
-              && ((fs.spec.available_rule
-                   == features::feature_spec::available_policy::always
-              && effective_version >= fs.spec.require_version) || (
-                    fs.spec.available_rule == features::feature_spec::available_policy::new_clusters_only
-                    &&
-                    original_version >= fs.spec.require_version
-                    ))) {
+            if ((fs.get_state() == features::feature_state::state::unavailable ||
+           fs.get_state() == features::feature_state::state::available) &&
+          ((fs.spec.available_rule ==
+                features::feature_spec::available_policy::always &&
+            effective_version >= fs.spec.require_version) ||
+           (fs.spec.available_rule ==
+                features::feature_spec::available_policy::new_clusters_only &&
+            original_version >= fs.spec.require_version))) {
```

Comment:
@BenPope @michael-redpanda did this condition change in more ways than formatting? i don't quite trust my mental parsing...